src/main/java/org/kuali/kfs/sys/businessobject/Country.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * The Kuali Financial System, a comprehensive financial management system for higher education.
+ *
+ * Copyright 2005-2021 Kuali, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.kuali.kfs.sys.businessobject;
+
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectBase;
+import org.kuali.kfs.core.api.mo.common.active.MutableInactivatable;
+
+public class Country extends PersistableBusinessObjectBase implements MutableInactivatable {
```

Comment:
it looks like this overlay way added to add the second constructor.  I only see the second constructor used in unit tests.  I am wondering if it would make sense to remove this file from the overlay and just use the setters in the unit test